serve/mlc_serve/logging_utils.py

```diff
@@ -0,0 +1,129 @@
+"""Setup structed logging."""
+import logging
+import sys
+from typing import List, Dict, Tuple, Any
+
+import structlog
+from structlog.types import EventDict
+from structlog.typing import Processor, WrappedLogger, EventDict
+
+_JSON = False
+
+def json_logging_enabled() -> bool:
+    global _JSON
+    return
```

Comment:
this appears unused

Reply:
I kind of hacked this into the parent process, so that we could mirror the right settings, when we move to octopy we  can clean up.

Reply:
For simplicity will keep the logging code the same between these if you guys are OK.